index.bs

```diff
@@ -409,6 +409,39 @@ model]]:
 
 1. Run [=record a user activation=] given <var ignore>document</var>.
 
+<h4 id="bounce-tracking-mitigations-web-authentication-monkey-patch">Web Authentication Monkey Patch</h4>
+
+A successful credential access, using the Web Authentication API, is also treated as a user activation for bounce-tracking purposes.
+
+Add the following argument to the [Discover From External Source](https://www.w3.org/TR/webauthn-2/#sctn-discover-from-external-source) algorithm:
```

Comment:
Sorry, I just noticed you are using the "TR" version of the spec.  Generally we prefer the editors draft:

https://w3c.github.io/webauthn/

Reply:
Done.